playbooks/roles/rbenv/tasks/main.yml

```diff
@@ -162,6 +162,14 @@
   tags:
     - install
     - install:base
+	
```

Comment:
Can you remove this "\t". Travis Build is failing because of this
========= travis logs begin===========
found character '\t' that cannot start any token
  in "./playbooks/roles/rbenv/tasks/main.yml", line 165, column 1
========= travis logs end===========